Resources/views/standard_layout.html.twig

```diff
@@ -142,7 +142,13 @@ file that was distributed with this source code.
                                                         {% if not loop.last  %}
                                                             <li>
                                                                 {% if menu.uri is not empty %}
-                                                                    <a href="{{ menu.uri }}">{{ menu.label|raw }}</a>
+                                                                    <a href="{{ menu.uri }}">
+                                                                        {% if menu.extra('safe_label', false) %}
```

Comment:
I think `safe_label` is good, because it is consistent with https://github.com/KnpLabs/KnpMenuBundle/blob/5e7c44cb084b5c4f842f94ab443fb5dbb3f7a44c/Resources/views/menu.html.twig So really, LGTM.